### PR TITLE
Pacman 5.0.2 proxy maintenance (including version bump).

### PR DESCRIPTION
**Pull request by holgersson32644**

### CODE REVIEW

File: sys-apps/pacman/pacman-5.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+PYTHON_COMPAT=( python2_7 )
+
+DESCRIPTION="Archlinux's binary package manager"
+HOMEPAGE="https://archlinux.org/pacman/"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.archlinux.org/pacman.git"
+	EGIT_BRANCH="master"
```
Comment (Coacher) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: EGIT_BRANCH seems to be redundant here.

File: sys-apps/pacman/pacman-5.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	# https://www.archlinux.org/news/phasing-out-i686-support/
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="curl debug doc +gpg"
+
+COMMON_DEPEND="app-arch/libarchive[lzma]
```
Comment (Coacher) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: SLOT operator needed.

File: sys-apps/pacman/pacman-5.0.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+	gpg? ( >=app-crypt/gpgme-1.4.0 )
```
Comment (Coacher) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: Same.

File: sys-apps/pacman/pacman-5.0.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+	dev-libs/openssl:=
```
Comment (Coacher) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: You should specify "0" SLOT explicitly. See openssl info on slots in metadata.

File: sys-apps/pacman/pacman-5.0.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+	curl? ( net-misc/curl )
+	sys-devel/gettext
```
Comment (Coacher) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: gettext required at runtime looks suspicious. If it's really needed a comment in the build would help.

File: sys-apps/pacman/pacman-5.0.2.ebuild [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+	# is passed:
+	sed -i -e '/-Werror/d' configure.ac || die
+
+	default
+}
+
+src_configure() {
+	local myeconfargs=(
+	    --localstatedir=/var
+		--disable-git-version
```
Comment (Coacher) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: Inconsistent indent.

File: sys-apps/pacman/metadata.xml [hunk 7]
```diff
@@ -1,7 +1,21 @@
...
+	<maintainer type="person">
+		<email>holgersson@posteo.de</email>
+		<name>Nils Freydank</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<use>
+		<flag name="doc">Install extended documentation using <pkg>app-doc/doxygen</pkg>. (Man pages are included by default.)</flag>
```
Comment (Coacher) [inline sys-apps/pacman/metadata.xml]: Why default global doc USE description doesn't fit here?

File: sys-apps/pacman/pacman-5.0.2.ebuild (hunk 1 above)
Comment (holgersson32644) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: I thought of easier maintenance if the branch might be changed later. Removed it locally (git push pending).

File: sys-apps/pacman/pacman-5.0.2.ebuild (hunk 2 above)
Comment (holgersson32644) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: Fixed locally (:=)

File: sys-apps/pacman/pacman-5.0.2.ebuild (hunk 5 above)
Comment (holgersson32644) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: You’re right. Fixed.

File: sys-apps/pacman/pacman-5.0.2.ebuild (hunk 6 above)
Comment (holgersson32644) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: vim aligned tabs and spaces, sorry that I oversaw this.

File: sys-apps/pacman/metadata.xml (hunk 7 above)
Comment (holgersson32644) [inline sys-apps/pacman/metadata.xml]: In my opinion the global description is a bit vague.

File: sys-apps/pacman/pacman-5.0.2.ebuild (hunk 4 above)
Comment (holgersson32644) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: Fixed.

File: sys-apps/pacman/pacman-5.0.2.ebuild (hunk 3 above)
Comment (holgersson32644) [inline sys-apps/pacman/pacman-5.0.2.ebuild]: Fixed.

File: sys-apps/pacman/metadata.xml (hunk 7 above)
Comment (Amynka) [inline sys-apps/pacman/metadata.xml]: I think it is always better to make sure users understand what they enable by the flag. So I agree with this.

**Pull request closed without merging**